erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -635,3 +637,11 @@ def update_common_item_properties(item, location):
 	item.serial_no = location.serial_no
 	item.batch_no = location.batch_no
 	item.material_request_item = location.material_request_item
+
+
+def update_stock_entry_items(location, stock_entry):
+	"""Append Item Row in Stock Entry !"""
+	item = frappe._dict()
+	update_common_item_properties(item, location)
+	stock_entry.append('items', item)
```

Comment:
@shantanu-kadam-epi it won't create stock entry it will give error At least one warehouse is mandatory because the function you called that required warehouse to create stock entry.

Reply:
Updated code for the same